### DISCUSSION TITLE
Where is options for date column type?

### DISCUSSION THREAD

**Discussion by emejibka** [nhibernate/nhibernate-core#2788] [Q&A]
Hello.

I have some issue with sqlite dialect/provider - where is options for truncating time part from date parameters ?
We have application with oracle connection. In some map we need to use `CustomType("date")` bacause we have datetime column otherwise we have performance problem by parameters type mismatch. 
But in unit tests with sqlite we have some issue - time part of DateTime is truncated.
Where is the settings for sqlite to truncate time part of DateTime for date type?

**Comment by gliljas** [comment]
> time part of DateTime is truncated

Isn't that the idea when using "date"?

**Reply by emejibka** [reply]
Not in Oracle :) In Oracle date type can contrains hours, minutes and seconds. Difference with timestamp in milliseconds.

**Reply by hazzik** [reply]
#741
